requirements.txt

```diff
@@ -1,55 +1 @@
-alembic==0.9.2
-alembic-autogenerate-enums==0.0.2
-amqp==2.2.1
-bcrypt==3.1.3
-beautifulsoup4==4.6.0
-billiard==3.5.0.3
-blinker==1.4
-celery==4.1.0
-certifi==2017.4.17
-cffi==1.10.0
-chardet==3.0.4
-click==6.7
-contextlib2==0.5.5
-enum-compat==0.0.2
-enum34==1.1.6
-Flask==0.12.2
-Flask-Bcrypt==0.7.1
-Flask-Cors==3.0.3
-Flask-Login==0.4.0
-Flask-Migrate==2.0.4
-Flask-Script==2.0.5
-Flask-SQLAlchemy==2.2
-google-api-python-client==1.6.2
-greenlet==0.4.12
-gunicorn==19.7.1
-httplib2==0.10.3
-idna==2.5
-itsdangerous==0.24
-Jinja2==2.9.6
-kombu==4.1.0
-Mako==1.0.6
-MarkupSafe==1.0
-nameparser==0.5.3
-oauth2client==4.1.2
-oauthlib==2.0.2
-psycopg2==2.7.1
-pyasn1==0.3.1
-pyasn1-modules==0.0.10
-pycparser==2.17
-python-dateutil==2.6.0
-python-editor==1.0.3
-pytz==2017.2
-raven==6.2.1
-redis==2.10.5
-requests==2.18.1
-requests-oauthlib==0.8.0
-rsa==3.4.2
-six==1.10.0
-SQLAlchemy==1.1.11
-twython==3.5.0
-Unidecode==0.4.21
-uritemplate==3.0.0
-urllib3==1.21.1
-vine==1.1.4
-Werkzeug==0.12.2
+-r requirements/prod.txt
```

Comment:
@rahulbot -- I'd like to separate out the requirements into local and production at the very list as it doesn't make sense to install `pylint` in prod.